features/steps/steps.rb

```diff
@@ -172,3 +170,39 @@ def get_version_number(file_path)
   Maze.check.not_include(run_output, "[INFO]")
   Maze.check.not_include(run_output, "[DEBUG]")
 end
+
+Before('@installation') do
+  @base_dir = Dir.pwd
+  @output = `npm pack`
+  @bugsnag_cli_package_path = "#{@base_dir}/#{@output}"
+end
+
+When('I install the bugsnag-cli via {string} in a new directory') do |package_manager|
+  case package_manager
+  when 'npm'
+    @fixture_dir = "#{@base_dir}/features/cli/fixtures/#{package_manager}"
+    Dir.mkdir(@fixture_dir)
+    Dir.chdir(@fixture_dir)
+    @init_output = `npm init -y`
+    @install_output = `npm install #{@bugsnag_cli_package_path}`
+    Dir.chdir(@base_dir)
+  when 'yarn'
+    @fixture_dir = "#{@base_dir}/features/cli/fixtures/#{package_manager}"
+    Dir.mkdir(@fixture_dir)
+    Dir.chdir(@fixture_dir)
+    @init_output = `yarn init -y`
+    @install_output = `yarn add #{@bugsnag_cli_package_path}`
+    Dir.chdir(@base_dir)
+  when 'pnpm'
+    @fixture_dir = "#{@base_dir}/features/cli/fixtures/#{package_manager}"
+    Dir.mkdir(@fixture_dir)
+    Dir.chdir(@fixture_dir)
+    @init_output = `pnpm init -y`
+    @install_output = `pnpm add #{@bugsnag_cli_package_path}`
+    Dir.chdir(@base_dir)
+  end
+end
+
+Then('the {string} directory should contain {string}') do |directory, package|
+  Maze.check.include(`ls #{@fixture_dir}/#{directory}`, package)
```

Comment:
You're not actually using the directory name

Reply:
As far as I can tell I'm using the directory name. 

`Maze.check.include(`ls #{@fixture_dir}/#{directory}`, package)` translates to `Maze.check.include(`ls features/cli/fixtures/npm/node_modules/.bin/`, bugsnag-cli)`

Reply:
Sorry - not sure what I was thinking. I'm sure it still had `node_modules` in it when I wrote that comment 🤦